packages/page/src/Infrastructure/Symfony/HttpKernel/SuluPageBundle.php

```diff
@@ -568,6 +570,8 @@ public function prependExtension(ContainerConfigurator $container, ContainerBuil
                                 'list' => 'sulu_page.get_pages',
                                 'detail' => 'sulu_page.get_page',
                             ],
+                            'security_class' => Page::class,
```

Comment:
Would this also work to use the PageInterface? Not sure how this works when e.g. the page is extended by another custom implementation @alexander-schranz